examples/robot_test_suite/test_uds/demo_uds.robot

```diff
@@ -12,3 +12,17 @@ Go in default session
     ${response} =  Send uds raw    \x10\x01    uds_aux
 
     Log  ${response}
+
+Use tester present sender
+    [Documentation]    If no communication is exchanged with the client for more than 5
+    ...                seconds the control unit automatically exits the current session and
+    ...                returns to the "Default Session" back, and might go to sleep mode.
+    ...
+    ...                To avoid this issue if test steps take too long between uds commands,
+    ...                the tester present sender can be use. It will send
+    ...                at a define period a Tester Present, to signal to the device that
```

Comment:
```suggestion
    ...                at a defined period a Tester Present, to signal to the device that
```

Reply:
I corrected all those errors, sorry for that